.github/workflows/test-linux-cpu.yml

```diff
@@ -54,4 +54,6 @@ jobs:
 
         # Run Tests
         python3 -m torch.utils.collect_env
+        rm -rf torchvision/prototype
+        rm test/*prototype*
```

Comment:
This comes from 67fdc6d1b9ecd9bf955af86da60fa749bb05fb7f that should be reverted. With this we can check if I didn't screw up anything and branch cut can happen smoothly